src/Sylius/Bundle/WebBundle/Resources/views/Backend/Order/macros.html.twig

```diff
@@ -97,10 +97,14 @@
                 {% include 'SyliusWebBundle:Backend/Order:_paymentState.html.twig' %}
             </td>
             <td style="width: 20%">
-                {{ order.customer.fullname }}<br>
-                <a href="{{ path('sylius_backend_customer_show', {'id': order.customer.id}) }}">
-                    {{ order.customer.email }}
-                </a>
+                {% if order.customer is not null %}
```

Comment:
The order should always have customer, why we need this if?

I recall some dashboard issue, but that's probably query problem. We should always query for placed orders, not carts (which it the only possible order representation that could not have customer assigned yet).